meteor_packages/mats-common/templates/graph/graph.js

```diff
@@ -1932,68 +1932,102 @@ Template.graph.events({
     'click #axisSubmit': function (event) {
         event.preventDefault();
         var plotType = Session.get('plotType');
+        var axesCollapsed = Session.get('axesCollapsed');
         var changeYScaleBack = false;
         var newOpts = {};
         // get input axis limits and labels
         $("input[id^=x][id$=AxisLabel]").get().forEach(function (elem, index) {
             if (elem.value !== undefined && elem.value !== "") {
-                newOpts['xaxis' + (index === 0 ? "" : index + 1) + '.title'] = elem.value;
+                if (!axesCollapsed || index === 0) {
+                    // if we've collapsed the axes we only want to process the first one
+                    newOpts['xaxis' + (index === 0 ? "" : index + 1) + '.title'] = elem.value;
+                }
```

Comment:
Being unfamiliar with the code - the reason we have `axis` and `axes` in here is because there are multiple y/x-axis on a plot and the plural of axis is axes?

Reply:
Yep, variables with "axes" is referring to multiple axis-es. "Axis" is just one.

Reply:
Gotcha. Thanks, Molly!